frontend/src/component/segments/SegmentProjectAlert.tsx

```diff
@@ -0,0 +1,90 @@
+import { Alert, styled } from '@mui/material';
+import { formatEditStrategyPath } from 'component/feature/FeatureStrategy/FeatureStrategyEdit/FeatureStrategyEdit';
+import { IProjectCard } from 'interfaces/project';
+import { IFeatureStrategy } from 'interfaces/strategy';
+import { Link } from 'react-router-dom';
+import { formatStrategyName } from 'utils/strategyNames';
+
+const StyledAlert = styled(Alert)(({ theme }) => ({
+    marginTop: theme.spacing(1),
+}));
+
+interface ISegmentProjectAlertProps {
+    projects: IProjectCard[];
+    strategies: IFeatureStrategy[];
+    projectsUsed: string[];
+    availableProjects: IProjectCard[];
+}
+
+export const SegmentProjectAlert = ({
+    projects,
+    strategies,
+    projectsUsed,
+    availableProjects,
+}: ISegmentProjectAlertProps) => {
+    const projectList = (
+        <ul>
+            {Array.from(projectsUsed).map(projectId => (
+                <li key={projectId}>
+                    <Link to={`/projects/${projectId}`} target="_blank">
+                        {projects.find(({ id }) => id === projectId)?.name ??
+                            projectId}
+                    </Link>
+                    <ul>
+                        {strategies
+                            ?.filter(
+                                strategy => strategy.projectId === projectId
+                            )
+                            .map(strategy => (
+                                <li key={strategy.id}>
+                                    <Link
+                                        to={formatEditStrategyPath(
+                                            strategy.projectId!,
+                                            strategy.featureName!,
+                                            strategy.environment!,
+                                            strategy.id
+                                        )}
+                                        target="_blank"
+                                    >
+                                        {strategy.featureName!}{' '}
+                                        {formatStrategyNameParens(strategy)}
+                                    </Link>
+                                </li>
+                            ))}
+                    </ul>
+                </li>
+            ))}
+        </ul>
+    );
+
+    if (projectsUsed.length > 1) {
+        return (
+            <StyledAlert severity="info">
+                You can't specify a project for this segment because it is used
+                in multiple projects:
+                {projectList}
+            </StyledAlert>
+        );
+    }
+
+    if (availableProjects.length === 1) {
+        return (
+            <StyledAlert severity="info">
+                You can't specify a project other than{' '}
+                <strong>{availableProjects[0].name}</strong> for this segment
+                because it is used here:
```

Comment:
@thomasheartman @NicolaeUnleash we were a bit unsure of what text to use here (both this scenario and the one above), so if you have better suggestions they're welcome as always! ❤️

Reply:
does it pertain to this bit?

> You can't specify a different project if the segment is currently already in use in a project;

Reply:
nice, i guess this is ok for now, i will let Thomas to have the last word on the text :) 

- Please remove the project bullets and padding, it's enough if we have bullets for feature toggles
- what will happen if the list is very long? does the info has a max height and scroll inside?

Reply:
@thomasheartman `projectList` is the list of projects and features you see in the screenshot on the PR description where the segment is currently being used.

This specific message pertains to:
> You can't specify a different project if the segment is currently already in use in a project

While the one above in the code pertains to:
> You can't specify a project if the segment is currently in use in multiple projects

Thanks for the feedback, I think we'll go with the current text then 🙂 

@NicolaeUnleash Thanks, addressed your first point in https://github.com/Unleash/unleash/pull/3359/commits/da0e351070d4818696b3228099762ecd0c00067d - Hope it looks OK now:
![image](https://user-images.githubusercontent.com/14320932/226640601-eabd0b52-ef98-430c-b1b0-89c4d39290b8.png)

Regarding your second point, I didn't specify a max-height. My guess is that the behavior will be consistent with other places in Unleash: If the info alert grows, the modal will scroll if needed, which seems OK to me. If you'd like to specify a height, let me know the value and I can set it, otherwise I vote we leave it as is.